src/Components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+import './TodoList.scss';
+import { Todo } from '../../types';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = (props) => {
+  const { todos } = props;
+
+  return (
+    <div className="list">
+      {todos.map((todo: Todo) => (
+        <div
+          className="list__item"
+          key={todo.id}
+        >
+          <TodoInfo todo={todo} />
+        </div>
```

Comment:
here you are wrapping the TodoInfo into a div, but in the TodoInfo component you are using the fragment <></>. maybe move this div to the TodoInfo and get rid of the fragment?